Kickstarter-iOS/Features/UserBlockedBanner/UserBlockedBannerView.swift

```diff
@@ -0,0 +1,27 @@
+import Library
```

Comment:
Does this need custom UI beyond our normal banner? Why not use a `MessageBannerViewController`?

Reply:
the MessageBannerViewController UX is typically used to show a brief message that is dismissed after a few seconds. Messages like, "Your request was successful", etc. I wanted to have this user-blocked UI be static and didn't want to potentially confuse the UX